tests/implementations/test_base.py

```diff
@@ -15,7 +16,8 @@ def test_run_pipeline_deterministic_with_random_state(reducer, polis_convo_data)
 
     # Load test data
     loader = Loader(filepaths=[f"{fixture.data_dir}/votes.json"])
-    votes = loader.votes_data
+    votes =  preprocess_votes(loader.votes_data)
```

Comment:
If this works, that's awesome. But we should find the most complex test here (which will exercise the longest path involving statement_ids), copy it and rename to something like `test_run_pipeline_with_string_statement_ids()`

Reply:
The alternative is to find all the functions that operate on the statement_ids in their logic, and then for their tests, add a test involving stringified IDs.

But a coarse high-level test like you've done here is the quickest way.

the catch is that in theory some of the underlying processes could get messed up, and it's not erroring, but values are somehow different (esp if there's a `sort` function somewhere that's now lexicographic instead of numeric). But I don't think we do that anywhere...!